src/xenia/cpu/ppc/ppc_translator.cc

```diff
@@ -55,7 +61,9 @@ PPCTranslator::PPCTranslator(PPCFrontend* frontend) : frontend_(frontend) {
   // Passes are executed in the order they are added. Multiple of the same
   // pass type may be used.
   if (validate) compiler_->AddPass(std::make_unique<passes::ValidationPass>());
-  compiler_->AddPass(std::make_unique<passes::ContextPromotionPass>());
+  if (!cvars::disable_context_promotion) {
+    compiler_->AddPass(std::make_unique<passes::ContextPromotionPass>());
+  }
   if (validate) compiler_->AddPass(std::make_unique<passes::ValidationPass>());
```

Comment:
You can skip the validation as well. It's pointless if you skip `ContextPromotionPass`.

```
  if (!cvars::disable_context_promotion) {
    if (validate) compiler_->AddPass(std::make_unique<passes::ValidationPass>());
    compiler_->AddPass(std::make_unique<passes::ContextPromotionPass>());
    if (validate) compiler_->AddPass(std::make_unique<passes::ValidationPass>());
  }
```